features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumApplicationScope.kt

```diff
@@ -67,9 +66,17 @@ internal class RumApplicationScope(
         )
     )
 
-    val activeSession: RumScope?
+    val activeSession: RumSessionScope?
         get() {
-            return childScopes.find { it.isActive() }
+            val activeSessions = childScopes.filter { it.isActive() }
+            if (activeSessions.size > 1) {
+                sdkCore.internalLogger.log(
```

Comment:
is this related with this PR ?

Reply:
yes, it is. It is just a safeguard to inform that we have to make a selection from the multiple active sessions.